stylix/mk-target.nix

```diff
@@ -267,8 +269,5 @@ let
     };
 in
 {
-  imports = [
-    { options.stylix.targets.${name} = extraOptions; }
-    module
-  ];
+  imports = [ module ];
```

Comment:
Why are we switching back to `unionOfDisjoint` instead of using the module system?